ecms_base/features/custom/ecms_projects/config/install/core.base_field_override.taxonomy_term.project_category.description.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.project_category
+  module:
+    - text
+id: taxonomy_term.project_category.description
+field_name: description
+entity_type: taxonomy_term
+bundle: project_category
+label: Description
+description: ''
+required: false
+translatable: false
```

Comment:
This should be translatable.

Reply:
Good catch. I fixed this and also added the published, url alias, and status fields to match the other terms.